src/features/movies/MovieList/index.js

```diff
@@ -18,6 +18,7 @@ export const MovieList = () => {
 
   return (
     <>
+      <Header>Popular movies</Header>
```

Comment:
I think it will be better here to rename as Title. And create title in  a folder common. Еhen use it here. Because we have a similar title on the pages "Movie" and "People" , only the color changes. What do you think?

Reply:
You have right! I'll correct it 🙂